### PR TITLE
PR for Asgardeo configuration

### PR DESCRIPTION
**Pull request by DINUSHA26**

### CODE REVIEW

File: src/pages/Websites.js [hunk 1]
```diff
@@ -5,12 +5,20 @@ import axios from "axios";
...
 
 function Websites() {
   const [websites, setWebsites] = useState([]);
-  const [userId, setUserId] = useState(1000);
+  const [userId, setUserId] = useState("");
   const [isOpenWebsiteDeleteDialog, setIsOpenWebsiteDialog] = useState(false);
   const [deletingWebsite, setDeletingWebsite] = useState();
+  const { getDecodedIDToken } = useAuthContext();
+
+  getDecodedIDToken().then((decodedIDToken) => {
```
Comment (pavinduLakshan) [inline src/pages/Websites.js:17]: Let's do this inside a `useEffect`.

Comment (DINUSHA26) [reply src/pages/Websites.js]: fixed

File: src/pages/Layout.js [hunk 2]
```diff
@@ -24,187 +24,191 @@ import ExitToAppTwoToneIcon from "@mui/icons-material/ExitToAppTwoTone";
...
 
 function Layout(props) {
-  const { window } = props;
-  const [mobileOpen, setMobileOpen] = React.useState(false);
-  const { state, signOut } = useAuthContext();
-  const name = state.displayName;
+    const { window } = props;
+    const [mobileOpen, setMobileOpen] = React.useState(false);
+    const { state, signOut } = useAuthContext();
+    const name = state.displayName;
```
[the remark this answers is not in the document]
Comment (DINUSHA26) [reply src/pages/Layout.js:33]: fixed

File: src/pages/Layout.js [hunk 3]
```diff
@@ -24,187 +24,191 @@ import ExitToAppTwoToneIcon from "@mui/icons-material/ExitToAppTwoTone";
...
-    </div>
-  );
+    const drawer = (
+        <div>
+            <Toolbar sx={{ display: "flex", justifyContent: "center" }}>
+                <img src={Logo} alt="Logo-Img" style={{ height: "60px" }} />
+            </Toolbar>
+            <Divider />
+            <List>
+                {drawerItems.map((object) => (
```
[the remark this answers is not in the document]
Comment (DINUSHA26) [reply src/pages/Layout.js]: fixed

File: package.json [hunk 4]
```diff
@@ -3,7 +3,6 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@asgardeo/auth-react": "^2.0.5",
```
[the remark this answers is not in the document]
Comment (DINUSHA26) [reply package.json]: fixed , commit no :-e49feda

**Review by nimeshnuwanga — APPROVED** [review]

**Pull request merged**